terraform/app.tf

```diff
@@ -0,0 +1,30 @@
+# Create the web app.
+
+resource "cloudfoundry_app" "beis-roda-app" {
```

Comment:
Do we need to define what to use as a health check for the web server?

Reply:
we can do it. it defaults to /

Reply:
I've changed this to /health_check